REQUIRE

```diff
@@ -1 +1 @@
-julia 0.3
```

Comment:
The algorithm requires computing trace(A'*B) - i.e a matrices inner product. There is a BLAS routine that does that: vecdot(). However - apparently it was added in v0.4. I don't know how to hack this "nicely" without vecdot. sum(sum(A.*B)) is not "nice". I don't think anyone is using v0.3 anyway because the julia guys don't support this.